.github/workflows/release.yml

```diff
@@ -0,0 +1,129 @@
+name: CLI Release
+
+on:
+  workflow_dispatch:
+  # schedule:
+  #   - cron: '0 12 * * 2'  # Runs every Tuesday at 12:00 PM (noon) ET
+
+jobs:
+
+  checks:
+    runs-on: ubuntu-latest
+    outputs:
+      HAS_TAG: ${{ steps.check_tag.outputs.has_tag }}
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
```

Comment:
You're using git describe tags - so you'll need to add checkout depth 0 to this.